happy_bank_core/logic/registration.py

```diff
@@ -0,0 +1,28 @@
+"""Python module that contains Registration class"""
+import random
+import logging
+
+from happy_bank_core.logic.account import Account
+
+logger = logging.getLogger(__name__)
+
+
+class Registration:
+    """Registration class that contains register method"""
+
+    @staticmethod
+    def register(data: dict) -> Account:
+        """register method that gets dict and returns Account obj"""
+        try:
+            if len(data["fullname"]) == 0:
```

Comment:
If you need to verify that a statement is None or empty, then it's enough to use 'not' operator.

```suggestion
            if not data["fullname"]:
```